diff --git a/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUAPIDemoBar.m b/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUAPIDemoBar.m
index 12b27cb58bfeb93c95a05fcca98f018436f08ac2..7a161a68015b52d57c35e4280fc37ce84de12678 100644
--- a/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUAPIDemoBar.m
+++ b/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUAPIDemoBar.m
@@ -13,6 +13,7 @@
 #import "FUSquareButton.h"
 #import "FUManager.h"
 #import <SVProgressHUD.h>
+#import "FUHelpers.h"
 
 @interface FUAPIDemoBar ()<FUFilterViewDelegate, FUBeautyViewDelegate>
 
diff --git a/FaceUnity/FULiveDemo/Helpers/FUCamera.m b/FaceUnity/FULiveDemo/Helpers/FUCamera.m
index eb21ec856ef8cc5d03557c30c96f6f0a2cd04f30..f51ace2d9adab64927b222c8e37259922d428b26 100755
--- a/FaceUnity/FULiveDemo/Helpers/FUCamera.m
+++ b/FaceUnity/FULiveDemo/Helpers/FUCamera.m
@@ -10,6 +10,7 @@
 #import <UIKit/UIKit.h>
 #import "FURecordEncoder.h"
 #import <SVProgressHUD/SVProgressHUD.h>
+#import "FUHelpers.h"
 
 
 
diff --git a/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUBeautyView.m b/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUBeautyView.m
index 4ea4781b4040b018d2ef7ce54f4df7254dde92eb..50ae6b8fbf043a48fc50d234e19f7ef6e5b71d72 100644
--- a/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUBeautyView.m
+++ b/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUBeautyView.m
@@ -9,6 +9,7 @@
 #import "FUBeautyView.h"
 #import "UIImage+demobar.h"
 #import "UIColor+FUAPIDemoBar.h"
+#import "FUHelpers.h"
 
 @interface FUBeautyView ()<UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout>
 @end
diff --git a/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUFilterView.m b/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUFilterView.m
index 03057860315a0007ae9499c6e0341b0e29cd70af..a6a60fdcffbca34e6c009c98e80a1b3dc6a01c45 100644
--- a/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUFilterView.m
+++ b/FaceUnity/FULiveDemo/Modules/Beauty/FUAPIDemoBar/FUFilterView.m
@@ -9,6 +9,7 @@
 #import "FUFilterView.h"
 #import "UIColor+FUAPIDemoBar.h"
 #import "UIImage+demobar.h"
+#import "FUHelpers.h"
 
 @interface FUFilterView ()<UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout>
 
