diff --git a/FaceUnity/FULiveDemo/Helpers/FUManager.m b/FaceUnity/FULiveDemo/Helpers/FUManager.m
index 1e025f05ac6e440230fcdcadbba8f5b19d4faba4..65bbed89e6cd2065a61d88d91696c38782444aa0 100755
--- a/FaceUnity/FULiveDemo/Helpers/FUManager.m
+++ b/FaceUnity/FULiveDemo/Helpers/FUManager.m
@@ -8,7 +8,7 @@
 
 #import "FUManager.h"
 #import "FURenderer.h"
-#import "authpack.h"
+#import "FUAuthData.h"
 #import "FULiveModel.h"
 #import <sys/utsname.h>
 #import <CoreMotion/CoreMotion.h>
@@ -61,7 +61,7 @@ static FUManager *shareManager = NULL;
         
         /**这里新增了一个参数shouldCreateContext，设为YES的话，不用在外部设置context操作，我们会在内部创建并持有一个context。
          还有设置为YES,则需要调用FURenderer.h中的接口，不能再调用funama.h中的接口。*/
-        [[FURenderer shareRenderer] setupWithDataPath:path authPackage:&g_auth_package authSize:sizeof(g_auth_package) shouldCreateContext:YES];
+        [[FURenderer shareRenderer] setupWithDataPath:path authPackage:FUGetAuthData() authSize:FUGetAuthDataLength() shouldCreateContext:YES];
         
         /* 加载AI模型 */
         [self loadAIModle];
