diff --git a/FaceUnity/FULiveDemo/Helpers/FUManager.m b/FaceUnity/FULiveDemo/Helpers/FUManager.m
index 47792c731b28b254dc1bc81afb369451292e29a1..1d21c42d389d0b1464ba9bc1ca982cef1f2e99ce 100755
--- a/FaceUnity/FULiveDemo/Helpers/FUManager.m
+++ b/FaceUnity/FULiveDemo/Helpers/FUManager.m
@@ -8,7 +8,7 @@
 
 #import "FUManager.h"
 #import "FURenderer.h"
-#import "authpack.h"
+#import "FUAuthData.h"
 #import "FULiveModel.h"
 #import <sys/utsname.h>
 #import <CoreMotion/CoreMotion.h>
@@ -67,7 +67,7 @@ static FUManager *shareManager = NULL;
         
         CFAbsoluteTime startTime = CFAbsoluteTimeGetCurrent();
 
-        [[FURenderer shareRenderer] setupWithData:nil dataSize:0 ardata:nil authPackage:&g_auth_package authSize:sizeof(g_auth_package) shouldCreateContext:YES];
+        [[FURenderer shareRenderer] setupWithData:nil dataSize:0 ardata:nil authPackage:FUGetAuthData() authSize:FUGetAuthDataLength() shouldCreateContext:YES];
 
         CFAbsoluteTime endTime = (CFAbsoluteTimeGetCurrent() - startTime);
         NSLog(@"---%lf",endTime);
diff --git a/FaceUnity/FULiveDemo/Helpers/FUManager.h b/FaceUnity/FULiveDemo/Helpers/FUManager.h
index e64d994429711e59cb5e9715dc0f77443aff008b..ce48219a32f6575540a16e4e903d40085bb57fe9 100755
--- a/FaceUnity/FULiveDemo/Helpers/FUManager.h
+++ b/FaceUnity/FULiveDemo/Helpers/FUManager.h
@@ -9,7 +9,6 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 #import <AVFoundation/AVFoundation.h>
-#import "FURenderer.h"
 #import "FUBeautyParam.h"
 
 @class FULiveModel;
