diff --git a/FaceUnity/FULiveDemo/Helpers/FUManager.m b/FaceUnity/FULiveDemo/Helpers/FUManager.m
index ad307128aa2bd8ee713c284f1a881dcbc4afd4f8..89c2193823e61ef6a97b0808b0e718ed1bc0fd7f 100755
--- a/FaceUnity/FULiveDemo/Helpers/FUManager.m
+++ b/FaceUnity/FULiveDemo/Helpers/FUManager.m
@@ -8,7 +8,7 @@
 
 #import "FUManager.h"
 #import "FURenderer.h"
-#import "authpack.h"
+#import "FUAuthData.h"
 #import "FULiveModel.h"
 #import <sys/utsname.h>
 #import <CoreMotion/CoreMotion.h>
@@ -63,7 +63,7 @@ static FUManager *shareManager = NULL;
         
         CFAbsoluteTime startTime = CFAbsoluteTimeGetCurrent();
 
-        [[FURenderer shareRenderer] setupWithData:nil dataSize:0 ardata:nil authPackage:&g_auth_package authSize:sizeof(g_auth_package) shouldCreateContext:YES];
+        [[FURenderer shareRenderer] setupWithData:nil dataSize:0 ardata:nil authPackage:FUGetAuthData() authSize:FUGetAuthDataLength() shouldCreateContext:YES];
 
         CFAbsoluteTime endTime = (CFAbsoluteTimeGetCurrent() - startTime);
         
diff --git a/FaceUnity/FULiveDemo/Helpers/FUManager.h b/FaceUnity/FULiveDemo/Helpers/FUManager.h
index 7f47e48ee520156763389e99e5db694705ac312d..1d60ee0ca4ac4a50a66ecedb4715cd94f86d6b28 100755
--- a/FaceUnity/FULiveDemo/Helpers/FUManager.h
+++ b/FaceUnity/FULiveDemo/Helpers/FUManager.h
@@ -9,7 +9,6 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 #import <AVFoundation/AVFoundation.h>
-#import "FURenderer.h"
 #import "FUBeautyParam.h"
 
 @class FULiveModel;
